akka-bench-jmh-dev/src/main/scala/akka/BenchRunner.scala

```diff
@@ -0,0 +1,28 @@
+package akka
+
+import org.openjdk.jmh.results.RunResult
+import org.openjdk.jmh.runner.Runner
+import org.openjdk.jmh.runner.options.CommandLineOptions
+
+object BenchRunner extends App {
+  import scala.collection.JavaConversions._
+
+  val args2 = args.toList match {
+    case "quick" :: tail => "-i 1 -wi 1 -f1 -t1".split(" ").toList ::: tail
+    case "thorough" :: tail => "-i 10 -wi 10 -f1 -t1".split(" ").toList ::: tail
```

Comment:
I can see myself mis-smelling this a 100 times (is there a simpler word for it?):wink: